lib/ember_cli/path_set.rb

```diff
@@ -63,18 +67,19 @@ def build_error_file
     def bower
       @bower ||= begin
         bower_path = app_options.fetch(:bower_path) { which("bower") }
+        bower_executable = Pathname(bower_path.to_s)
```

Comment:
I don't remember exactly where I saw this syntax, but it came from [docs](https://ruby-doc.org/stdlib-2.1.2/libdoc/pathname/rdoc/Pathname.html#method-i-each_child).

It's definitely surprising, I'll replace it with `Pathname.new`.